### PR TITLE
Move revision

### PR DESCRIPTION
**Pull request by hariria**
### Description

Make Move landing page prettier, fix navigation title

### Checklist

- Do all Lints pass?
  - [x] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/sdks/ts-sdk.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ The TypeScript SDK allows you to connect, explore, and interact on the Aptos blo
 - [Source Code](https://github.com/aptos-labs/aptos-ts-sdk)
 - [Reference Docs](https://aptos-labs.github.io/aptos-ts-sdk/@aptos-labs/ts-sdk-latest)
 
-# Quickstart
+## Typescript Quickstart
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/sdks/ts-sdk.mdx:9]:
```suggestion
## TypeScript Quick Start
```

File: apps/nextra/pages/en/build/move/examples/first-move-module.mdx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Run the below command to compile the `hello_blockchain` module:
+
+```bash filename="Terminal"
+aptos move compile --named-addresses hello_blockchain=default
+```
+
+You will see output resembling:
+
+```text
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/move/examples/first-move-module.mdx:95]: This is technically `json`

File: apps/nextra/pages/en/build/move/examples/first-move-module.mdx [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+Now fund this account by running this command:
+
+```bash filename="Terminal"
+aptos account fund-with-faucet --account default
+```
+
+You will see output resembling:
+
+```text
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/move/examples/first-move-module.mdx:71]: similarly on json

File: apps/nextra/pages/en/build/move/examples/first-move-module.mdx [hunk 4]
```diff
@@ -0,0 +1,222 @@
+---
+title: "Your First Move Module"
+---
+
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/move/examples/first-move-module.mdx:4]: I thought these were removed?

**Review by gregnazario — APPROVED** [review]

File: apps/nextra/pages/en/build/sdks/ts-sdk.mdx (hunk 1 above)
Comment (hariria) [reply apps/nextra/pages/en/build/sdks/ts-sdk.mdx:9]: Haha I think it's one word actually, like https://cloud.google.com/shell/docs/quickstart

Comment (gregnazario) [reply apps/nextra/pages/en/build/sdks/ts-sdk.mdx:9]: At least `Typescript` -> `TypeScript`

File: apps/nextra/pages/en/build/move/examples/first-move-module.mdx (hunk 4 above)
Comment (hariria) [reply apps/nextra/pages/en/build/move/examples/first-move-module.mdx:4]: We don't have a single guide on how to write move for devs in the docs. I feel like we need at least one.

Comment (gregnazario) [reply apps/nextra/pages/en/build/move/examples/first-move-module.mdx:4]: Okay, then let's get this in... and then we'll have to update it.

File: apps/nextra/pages/en/build/sdks/ts-sdk.mdx (hunk 1 above)
Comment (hariria) [reply apps/nextra/pages/en/build/sdks/ts-sdk.mdx]: You got it

**Pull request merged**